src/utilities.jl

```diff
@@ -1,27 +1,27 @@
-####
-#### validation
-####
+log(message) = @info "$(now()) | $message"
```

Comment:
Why do you need this local shadow of `log` instead of just using `@info` directly?

Reply:
just to define the preferred structure `"$(now()) | $message"` in one callable place rather than re-spelling that everywhere

Reply:
Are you actually logging things that often?